### PR TITLE
[src] Add generate aws account external id route

### PR DESCRIPTION
**Pull request by caduri**

### CODE REVIEW

File: api/services/administration/general/schemas/setCloudCredentialsForAWS.yaml [hunk 1]
```diff
@@ -9,8 +9,3 @@ properties:
         description: Provide the IAM Role ARN connected to another AWS account 922761411349 and with the latest Spot Policy - https://docs.spot.io/administration/api/spot-policy-in-aws
         example: arn:aws:iam::1234567890:role/Spot_Iam_Role
         required: true
-      externalId:
```
Comment (sagishoshan) [inline api/services/administration/general/schemas/setCloudCredentialsForAWS.yaml:12]: we should keep this

File: api/services/administration/general/schemas/setCloudCredentialsForAWS.yaml [hunk 2]
```diff
@@ -9,8 +9,3 @@ properties:
...
-        type: string
-        description: Provide the External ID used while setting up the IAM Role. This should be a random string of your choice.
```
Comment (sagishoshan) [inline api/services/administration/general/schemas/setCloudCredentialsForAWS.yaml:14]: maybe just re-write the description?

**Review by sagishoshan — CHANGES_REQUESTED** [review]

File: api/services/administration/general/schemas/setCloudCredentialsForAWS.yaml (hunk 1 above)
Comment (caduri) [reply api/services/administration/general/schemas/setCloudCredentialsForAWS.yaml:12]: Now after the ui deployed its changes we can remove it

File: api/services/administration/general/schemas/setCloudCredentialsForAWS.yaml (hunk 2 above)
Comment (caduri) [reply api/services/administration/general/schemas/setCloudCredentialsForAWS.yaml:14]: Now after the ui deployed its changes we can remove it

**Review by sagishoshan — APPROVED** [review]

**Pull request merged**